job.go

```diff
@@ -181,7 +181,23 @@ func (j *Job) Weekday() (time.Weekday, error) {
 	if j.scheduledWeekday == nil {
 		return time.Sunday, ErrNotScheduledWeekday
 	}
-	return *j.scheduledWeekday, nil
+	return *j.scheduledWeekday[0], nil
+}
+
+// Weekdays returns a slice of time.Weekday that the Job will run in a week and
+// will return an error fi the Job is not shceduled weekly
+func (j *Job) Weekdays() ([]time.Weekday, error) {
+	var jobWeekDays []time.Weekday
+
+	if j.scheduledWeekday == nil {
+		jobWeekDays = append(jobWeekDays, time.Sunday)
```

Comment:
Out of curiosity, why do we append Sunday and return the slice plus the err here? If the job is not scheduled for a weekly basis maybe we could get a nil slice.

Reply:
I may be wrong about this, but this error was never called before. To calculate the [remaining days](https://github.com/go-co-op/gocron/blob/master/scheduler.go#L233) the scheduler just use the job attribute `scheduledWeekday`. I was reluctant to change this piece but that might a good idea. Will not break the current behavior.

Reply:
*just for test purposes

Reply:
Just let the `Weekday()` with this berravior. Done!